webapp-django/crashstats/symbols/tests/test_views.py

```diff
@@ -67,6 +69,7 @@ def mocked_get_key(key_name):
                 if key_name in self.known_bucket_keys:
                     mocked_key = mock.Mock()
                     mocked_key.key = key_name
+                    mocked_key.content_type = 'application/binary-octet-stream'
```

Comment:
I don't know anymore. I tried to make sense of it during lunch but I can't find any place where it even tests what gets set in the mocked objects. I saw, just leave it as is. Don't let this comment block.